local.build.gradle

```diff
@@ -0,0 +1,8 @@
+android {
```

Comment:
This file should be ignored.

Reply:
my bad, i put the local.build.gradle in the wrong directory, i deleted it